lsr_role2collection.py

```diff
@@ -432,7 +432,7 @@ def vars_cb(self, a_item, ru_item):
         """handle vars of Ansible item, or vars from a vars file"""
         for var in a_item.get("vars", []):
             logging.debug(f"\tvar = {var}")
-            if f"{var}" == "roletoinclude":
+            if var == "roletoinclude":
                 lsr_rolename = self.src_owner + "." + self.rolename
                 if a_item["vars"][f"{var}"] == lsr_rolename:
                     ru_item["vars"][f"{var}"] = self.prefix + self.rolename
```

Comment:
these two lines also - `f"{var}"` -> `var`

Reply:
Sorry, I should have tried them... (;_;) Yes, no need to convert to a string there, too.
Replaced and squashed.